config.example.wikimedia.yaml

```diff
@@ -117,36 +117,41 @@ templates:
                 request:
                   uri: http://graphoid.wikimedia.org/{domain}/v1/png/{title}/{revision}/{graph_id}
 
-      /{module:mobileapps}/v1/html/{title}:
-        get:
-          x-request-handler:
-            - get_from_backend:
-                request:
-                  uri: http://appservice.wmflabs.org/{domain}/v1/page/mobile-html/{title}
-      /{module:mobileapps}/v1/sections/{title}:
-        get:
-          x-request-handler:
-            - get_from_backend:
-                request:
-                  uri: http://appservice.wmflabs.org/{domain}/v1/page/mobile-html-sections/{title}
-      /{module:mobileapps}/v1/sections-lead/{title}:
-        get:
-          x-request-handler:
-            - get_from_backend:
-                request:
-                  uri: http://appservice.wmflabs.org/{domain}/v1/page/mobile-html-sections-lead/{title}
-      /{module:mobileapps}/v1/sections-remaining/{title}:
-        get:
-          x-request-handler:
-            - get_from_backend:
-                request:
-                  uri: http://appservice.wmflabs.org/{domain}/v1/page/mobile-html-sections-remaining/{title}
-      /{module:mobileapps}/v1/text/{title}:
-        get:
-          x-request-handler:
-            - get_from_backend:
-                request:
-                  uri: http://appservice.wmflabs.org/{domain}/v1/page/mobile-text/{title}
+      /{module:mobileapps}/v1:
```

Comment:
Euh, why do we have `v1` in `/sys`? What's the point of that? I guess I should be able to answer that question, but can't :P Would you mind removing it and adjusting the paths in mediawiki/v1/mobileapps.yaml ?